File: pos/src/main/java/org/mini/project/pos/posservice/model/customer/Customer.java
package org.mini.project.pos.posservice.model.customer;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
import org.mini.project.pos.posservice.model.PaymentType;

import java.util.List;

@Setter
@Getter
@Builder
public class Customer {

  private long id;

  private String name;

  private String username;

  private String password;

  private List<PaymentType> paymentTypes;

  private List<Long> orderId;

  private Long balance;
}

File: customer/src/main/kotlin/com/miniproject/pos/customer/model/Customer.kt
package com.miniproject.pos.customer.model

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
data class Customer(

    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    val id: Long? = 0,
    val name: String?,
    val username: String?,
    val password: String?,
    val paymentType: MutableList<PaymentType>?,
    val orderId: MutableList<Long>? = mutableListOf()

)    {
    constructor() : this(0, "", "", "", mutableListOf(PaymentType.CASH), mutableListOf())
}